src/Dfe.PlanTech.Domain/Caching/Models/CacheKey.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Dfe.PlanTech.Domain.Caching.Models;
+
+internal static class CacheKey
+{
+    public static string Make(string name, string subname, params object[] items) => $"{name}.{subname}({string.Join(',', items.Select(x => x?.ToString()))})";
```

Comment:
Similar comment, are we planning to use this? No harm in leaving it but I think there are some unused things we can remove and trim down the integration